bundles/http/Citrix/2ec16a1b-9423-45a6-8fd8-b0c42c916c0f/cb5694f4-3740-45d6-9817-44d451edb2d9.js

```diff
@@ -0,0 +1,1136 @@
+}
+
+
+
+//Defining Actions Methods
+function createMeeting({dataStore, client, actionParameters}){
+    const request = client.fetchSync(`users/${actionParameters.Userid}/meetings`, {
+        method : "POST",
+        body : JSON.stringify({
+            topic:actionParameters.topic,
+            type : actionParameters.type,
+            start_time: actionParameters.start_time,
+            duration :actionParameters.duration,
+            password : actionParameters.password,
+            recurrence : {
+                type : actionParameters.r_type,
+                repeat_interval:actionParameters.repeat_interval,
+                weekly_days : actionParameters.weekly_days,
+                monthly_day : actionParameters.monthly_day,
+                end_times : actionParameters.end_times,
+            },
+            settings : {
+                alternative_hosts : actionParameters.alternative_hosts,
+                global_dial_in_countries : [actionParameters.us,actionParameters.mx,actionParameters.ar,actionParameters.au,actionParameters.ca]
+            }
+        })
+    })
+    if(request.ok){
+        let meeting = request.jsonSync();
+        // console.log(JSON.stringify(meeting.host_email));
+        syncMeetings(dataStore,client,[{meetingid:meeting.id,userid:actionParameters.Userid,email:meeting.host_email}]);
+        syncMeetingsDetails(dataStore,client);
+        syncMeetingInvitations(dataStore,client); 
+    }
+}
+
+//   Create Meeting Fucntion with two co-organizers
+
+function createMeetingtwoorganizers({dataStore, client, actionParameters}){
+    const request = client.fetchSync(`users/${actionParameters.Userid}/meetings`, {
+        method : "POST",
+        body : JSON.stringify({
+            topic:actionParameters.topic,
+            type : actionParameters.type,
+            start_time: actionParameters.start_time,
+            duration :actionParameters.duration,
+            password : actionParameters.password,
+            recurrence : {
+                type : actionParameters.r_type,
+                repeat_interval:actionParameters.repeat_interval,
+                weekly_days : actionParameters.weekly_days,
+                monthly_day : actionParameters.monthly_day,
+                end_times : actionParameters.end_times,
+            },
+            settings : {
+                alternative_hosts : `${actionParameters.alternative_hosts1},${actionParameters.alternative_hosts2}`,
+                global_dial_in_countries : [actionParameters.us,actionParameters.mx,actionParameters.ar, actionParameters.au,actionParameters.ca]
+            }
+        })
+    })
+    if(request.ok){
+        let meeting = request.jsonSync();
+        // console.log(JSON.stringify(meeting.host_email));
+        syncMeetings(dataStore,client,[{meetingid:meeting.id,userid:actionParameters.Userid,email:meeting.host_email}]);
+        syncMeetingsDetails(dataStore,client);
+        syncMeetingInvitations(dataStore,client);

```

Comment:
These 3 functions are always called in batch.
Why not create one universal function `syncMeetingsData()` which calls all these 3 functions?